Source/MediaInfo/Audio/File_Usac.h

```diff
@@ -268,9 +350,20 @@ public :
         int8u                       sampling_frequency_index;
         int8u                       coreSbrFrameLengthIndex;
         int8u                       baseChannelCount;
+        int8u                       stereoConfigIndex;
         bool                        IsNotValid;
         bool                        loudnessInfoSet_IsNotValid;
         bool                        harmonicSBR;
+        bool                        bs_interTes;
+        bool                        bs_pvc;
+        bool                        noiseFilling;
+        bool                        common_window;
+        bool                        common_tw;
+        bool                        tw_mdct;
+        arith_context               arithContext[2]; //TODO: resize per channel count

```

Comment:
Should be easy to resize depending on the real channel count (IIRC the channel count is already somewhere).